chart/registry-rewriter/templates/mutatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: {{ template "registry-rewriter.fullname" . }}
+  labels:
+    app: {{ template "registry-rewriter.name" . }}
+    chart: {{ template "registry-rewriter.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+webhooks:
+- clientConfig:
+    service:
+      namespace: kube-system
+      name: {{ template "registry-rewriter.fullname" . }}-service
+    caBundle: {{ .Values.caBundle }}
+  failurePolicy: Fail
+  name: {{ template "registry-rewriter.fullname" . }}-service.kube-system.svc
+  namespaceSelector:
+    matchExpressions:
+      - key: mutateimagename
+        operator: NotIn
+        values: [never]
```

Comment:
It will only apply things that either dont have `mutateimagename` label set or it is set to something other than `never` 

Because the deployment of the hook has `mutateimagename: never` it will be ignored.

Reply:
Nice